rfcs/interop_2023.md

```diff
@@ -0,0 +1,84 @@
+# RFC 116: Interop 2023
+
+## Summary
+
+Interop 2023 is an effort to improve interoperability in areas believed to be important to web developers and users. This is the next iteration of [Interop 2022](https://github.com/web-platform-tests/interop-2022), an effort which is still ongoing at the time of this RFC.
+
+The focus areas will be selected by a proposals period followed by a proposals review and selection process. The [Interop 2023 team](https://github.com/orgs/web-platform-tests/teams/interop-2023) will manage this process and make decisions based on consensus, see [governance](#governance).
+
+There will be a dashboard at https://wpt.fyi/interop-2023.
+
+## Details
+
+### Timeline
+
+During 2022:
+
+- September 15 to October 15: Proposals period.
+- October 16 to October 30: Proposal discussion/refinement. (New proposals no longer accepted.)
+- November 1 to November 30: Proposal selection. (The team must reach consensus no later than November 30.)
+- December 1 to December 16:
+  - Start review of tests. Write needed tests.
+  - Start involving PR teams. Write rough drafts of announcements. Begin creating dashboard.
+
+During 2023:
+
+- January 9 to January 20:
+  - Finalize test selection.
+  - Complete announcement plans. Prepare dashboard for launch.
+- January 23 to January 27: Launch.
+
+If necessary, additional changes to the test lists can be made throughout the year with the consensus of the team, see [updating the tests](#updating-the-tests).
+
+### Governance
+
+The [Interop 2023 team](https://github.com/orgs/web-platform-tests/teams/interop-2023) will manage the overall Interop 2023 effort. The team will be formed with the same initial membership as the [’22 team](https://github.com/orgs/web-platform-tests/teams/interop-2022).
+
+The team will meet regularly, with agenda issues created in the repo at least 48 hours in advance, and meeting minutes posted in the issue.
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no objections. Additionally, during the [proposal selection](#proposal-selection) process, the positions of implementers in each browser engine are used to determine whether a proposal will be accepted, to ensure that the proposal is on track to interoperability across browser engines.
+
+The Interop 2023 effort is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+The [WPT core team](https://github.com/orgs/web-platform-tests/teams/wpt-core-team/) is not expected to be involved beyond this RFC, but could in exceptional cases intervene via the RFC process to amend the governance.
```

Comment:
Is this necessary? This is implicit anyway.

Reply:
We can leave it implicit, I take it that's your preference.